From f854f1bb65bb66af0764a98e3a71f73a97decd25 Mon Sep 17 00:00:00 2001
From: Pat Thoyts <patthoyts@users.sourceforge.net>
Date: Fri, 10 Sep 2010 10:40:03 +0100
Subject: [PATCH] curl: treat .netrc files as text when opening to handle line endings

The .netrc file may well use CRLF on Windows and should be opened using
text mode to avoid reading in the CR as part of the authentication data.

Signed-off-by: Pat Thoyts <patthoyts@users.sourceforge.net>
---
 lib/netrc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/netrc.c b/lib/netrc.c
index 1c425fe..27d21bd 100644
--- a/lib/netrc.c
+++ b/lib/netrc.c
@@ -138,7 +138,7 @@ int Curl_parsenetrc(const char *host,
     netrc_alloc = TRUE;
   }
 
-  file = fopen(netrcfile, "r");
+  file = fopen(netrcfile, "rt");
   if(file) {
     char *tok;
     char *tok_buf;
-- 
1.7.2.1.msysgit.0

